inst/templates/monolix-step-01-preprocess.Rmd

```diff
@@ -0,0 +1,180 @@
+---
+title: "Monolix Data Processing"
+author: "Your Name"
+date: "Created: YYYY-MM-DD, Last run: `r Sys.Date()`"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+options(knitr.kable.NA = '-')
+
+library(tidyverse)
+library(glue)
+library(here)
+library(knitr)
+library(kableExtra)
+library(usethis)
+```
+
+## Install and check the data package
+
+<!-- check against data specs: https://github.com/FredHutch/VISC-Assay-Methods/tree/master/PK/Assay_Data_Specs -->
+
+```{r install-data-package, eval=FALSE}
+
+remotes::install_git("N:/cavd/studies/cvdxxx/pdata/xxx.git")
+```
+
+```{r load-data-package}
+
+# Load PK concentrations and assign to pk_data 
+library(Ho785)
+data("Ho785_covance_elisa_pk")
+
+pk_data <- Ho785_covance_elisa_pk
+
+glimpse(pk_data)
+```
+
+<!-- The STP should create a table of missing concentrations in the QC report. You may want to further investigate patterns of missingness here. -->
+
+```{r check-for-missing-concentrations}
+
+# this will only work if there is one row per expected visit in the data package
+pk_data %>% 
```

Comment:
In many of our human studies, `NA` implies BLOQ, so this needs to be checked for each protocol.

Reply:
Thanks! I added this in the HTML comments.